### ISSUE TITLE
Accessibility: Missing Label: Web Preview iFrame need title attribute

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#1994]
This issue has been split out from: https://github.com/Automattic/wp-calypso/issues/697

The web preview iframe needs a title attribute.

**Comment by SiobhyB** [comment]
@hafizrahman: I think it's best to stick with something simple like `Preview` on this as, I'm assuming, that will be easier for i18n than any variations.

<img width="1313" alt="screen shot 2015-12-29 at 19 34 45" src="https://cloud.githubusercontent.com/assets/2998162/12040739/6a47b84c-ae63-11e5-9907-32aefacb8c9b.png">

**Comment by amandarush** [comment]
By way of asking for clarification, why are we using a title attribute for the preview frame as opposed to using Aria? Is it so we can include the I18N, (not that I'm against including that at all)?

**Comment by SiobhyB** [comment]
Hi @amandarush! It was my assumption that the title attribute should be used for iframes, based on my reading here:

http://webaim.org/techniques/frames

We're open to learning more if that's not the best practice though! Is there an ARIA attribute that could be used and that would be better?

CCing @davidakennedy also, as this was initially reported by him.

**Comment by amandarush** [comment]
Hi,

In this case, as it's the preview window which is pulling content from possibly another site/page other than Calypso, this would be bets practice. Thanks for the clarification

**Comment by SiobhyB** [comment]
Thank you for confirming that @amandarush, and for all of the help you've already given around accessibility here too. :)
